src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -186,7 +185,18 @@ public virtual TextSpan GetDefaultCompletionListSpan(SourceText text, int caretP
         /// <param name="options">Completion options</param>
         /// <param name="displayOptions">Display options</param>
         /// <param name="cancellationToken"></param>
-        internal abstract Task<CompletionDescription?> GetDescriptionAsync(Document document, CompletionItem item, CompletionOptions options, SymbolDescriptionOptions displayOptions, CancellationToken cancellationToken = default);
+        internal virtual async Task<CompletionDescription?> GetDescriptionAsync(Document document, CompletionItem item, CompletionOptions options, SymbolDescriptionOptions displayOptions, CancellationToken cancellationToken = default)
+        {
+            var provider = GetProvider(item);
+            if (provider is null)
```

Comment:
can this happen?

Reply:
I don't believe it's possible if the item is created "properly" via our system, but theoretically someone could pass in an arbitrary item they created themselves. So we'd need this in place to guard that